drivers/ethernet/eth_dw.c

```diff
@@ -372,9 +373,16 @@ static struct eth_runtime eth_0_runtime = {
 #endif
 };
 
-static struct net_if_api api_funcs = {
-	.init = eth_initialize,
-	.send = eth_tx,
+static enum eth_hw_caps eth_dw_get_capabilities(struct device *dev)
```

Comment:
In https://github.com/zephyrproject-rtos/zephyr/pull/6601/commits/a20f59eea9194d7f95f8b79b7b1367f99eb089ac#diff-478140223e6b04c1b262b95c27afb803R124 , there's a code to treat caps as 0 if the function pointer is not set. Then why each and every driver is patched to include the empty function still?

Reply:
> Then why each and every driver is patched to include the empty function still?

Indeed, I will remove the empty func from the drivers as that is not needed.